src/main/java/org/openpnp/machine/reference/vision/wizards/ReferenceFiducialLocatorConfigurationWizard.java

```diff
@@ -0,0 +1,233 @@
+package org.openpnp.machine.reference.vision.wizards;
```

Comment:
No argument, I was not expecting this format to be perfect.  More beauty newlines coming right up.